src/app.vue

```diff
@@ -0,0 +1,748 @@
+                tabindex="-1" aria-labelledby="tourism-info-header">
+                <div class="offcanvas-header d-block">
+                    <button type="button" class="btn btn-link ps-0" data-bs-dismiss="offcanvas"
+                        aria-label="Close">Back</button>
+                    <h2 class="offcanvas-title" id="tourism-info-header">Tourism</h2>
+                </div>
+                <div class="offcanvas-body">
+                    <p>Monetize with expanding on efficient air infrastructure in area's where regular infrastructure is
+                        slow and expensive to maintain. Get ahead by deploying your own fleet and/or vertiports.</p>
+                    <ul>
+                        <li>Deploy your fleet really quickly and start monetizing this month</li>
+                        <li>Reach your target audience through a free for them to use application</li>
+                    </ul>
+                    <div class="infobox">
+                        <h4>Need help setting up ARROW for your aircraft or vertiport(s)?</h4>
+                        <p>We are happy to show you the possibilities, create a strategy and connect you to partners to make
+                            the onboarding process smooth for you. <a href="#" target="_blank">Book a call with us</a>
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <div class="offcanvas offcanvas-end" data-bs-backdrop="false" data-bs-scroll="true" id="agriculture-info"
+                tabindex="-1" aria-labelledby="agriculture-info-header">
+                <div class="offcanvas-header d-block">
+                    <button type="button" class="btn btn-link ps-0" data-bs-dismiss="offcanvas"
+                        aria-label="Close">Back</button>
+                    <h2 class="offcanvas-title" id="agriculture-info-header">Agriculture</h2>
+                </div>
+                <div class="offcanvas-body">
+                    <p>Monetize with expanding on efficient air infrastructure in area's where regular infrastructure is
+                        slow and expensive to maintain. Get ahead by deploying your own fleet and/or vertiports.</p>
+                    <ul>
+                        <li>Deploy your fleet really quickly and start monetizing this month</li>
+                        <li>Reach your target audience through a free for them to use application</li>
+                    </ul>
+                    <div class="infobox">
+                        <h4>Need help setting up ARROW for your aircraft or vertiport(s)?</h4>
+                        <p>We are happy to show you the possibilities, create a strategy and connect you to partners to make
+                            the onboarding process smooth for you. <a href="#" target="_blank">Book a call with us</a>
+                        </p>
```

Comment:
I suggest use this section to pitch to operators, while the text above to pitch to use-case specific customers.

```suggestion
                        <h4>Are you an asset owner/operator with aircraft or vertiport(s)?</h4>
                        <p>Deploy with us to avoid the complexity of scheduling and managing your assets.  <a href="#" target="_blank">Deploy and start earning</a>
                        </p>
```